Client/Ecosia/UI/EcosiaFindInPageBar.swift

```diff
@@ -0,0 +1,238 @@
+class EcosiaFindInPageBar: UIView {
+    
+    private lazy var searchView: UIView = {
+        let view = UIView()
+        view.layer.cornerRadius = (EcosiaFindInPageBar.barHeight - 2*EcosiaFindInPageBar.searchViewTopBottomSpacing)/2
+        return view
+    }()
+    private lazy var searchTextField: UITextField = {
+        let textField = UITextField()
+        textField.addTarget(self, action: #selector(didTextChange), for: .editingChanged)
+        textField.font = .preferredFont(forTextStyle: .body)
+        textField.placeholder = .localized(.findInPage)
+        textField.autocapitalizationType = .none
+        textField.autocorrectionType = .no
+        textField.inputAssistantItem.leadingBarButtonGroups = []
+        textField.inputAssistantItem.trailingBarButtonGroups = []
+        textField.enablesReturnKeyAutomatically = true
+        textField.returnKeyType = .search
+        textField.accessibilityIdentifier = "FindInPage.searchField"
+        textField.delegate = self
+        return textField
+    }()
+    private lazy var matchCountLabel: UILabel = {
+        let label = UILabel()
+        label.font = .preferredFont(forTextStyle: .footnote)
+        label.textAlignment = .right
+        label.isHidden = true
+        label.accessibilityIdentifier = "FindInPage.matchCount"
+        return label
+    }()
+    private lazy var previousButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: "find_previous")?.withRenderingMode(.alwaysTemplate), for: .normal)
+        button.isEnabled = false
+        button.accessibilityLabel = .FindInPagePreviousAccessibilityLabel
+        button.addTarget(self, action: #selector(didFindPrevious), for: .touchUpInside)
+        button.accessibilityIdentifier = "FindInPage.find_previous"
+        return button
+    }()
+    private lazy var nextButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: "find_next")?.withRenderingMode(.alwaysTemplate), for: .normal)
+        button.isEnabled = false
+        button.accessibilityLabel = .FindInPageNextAccessibilityLabel
+        button.addTarget(self, action: #selector(didFindNext), for: .touchUpInside)
+        button.accessibilityIdentifier = "FindInPage.find_next"
+        return button
+    }()
+    private lazy var closeButton: UIButton = {
+        let button = UIButton()
+        button.setTitle(.localized(.done), for: .normal)
+        button.accessibilityLabel = .FindInPageDoneAccessibilityLabel
+        button.addTarget(self, action: #selector(didPressClose), for: .touchUpInside)
+        button.accessibilityIdentifier = "FindInPage.close"
+        return button
+    }()
+    private lazy var topBorder = UIView()
+    
+    weak var delegate: EcosiaFindInPageBarDelegate?
+
+    private static let savedTextKey = "findInPageSavedTextKey"
+    private static let barHeight: CGFloat = 60
+    private static let searchViewTopBottomSpacing: CGFloat = 8
```

Comment:
I've seen that the codebase style utilised by Ecosia and FF is to add a struct `UX` at the top of the `UIView`/component that defines all these hardcoded values